src/Symfony/Component/HttpFoundation/Session/Storage/Handler/PdoSessionHandler.php

```diff
@@ -151,7 +151,7 @@ class PdoSessionHandler extends AbstractSessionHandler
      *
      * @throws \InvalidArgumentException When PDO error mode is not PDO::ERRMODE_EXCEPTION
      */
-    public function __construct(\PDO|string $pdoOrDsn = null, array $options = [])
+    public function __construct(#[\SensitiveParameter] \PDO|string $pdoOrDsn = null, array $options = [])
```

Comment:
The `$options` contains the password, ut should be sensitive too, as you did for PdoStore.

Reply:
yup, fixed